File: src/main/java/org/profit/server/NettyServer.java
package org.profit.server;

import io.netty.channel.Channel;
import io.netty.channel.ChannelInitializer;

/**
 * @author Kyia
 */
public interface NettyServer extends Server {

	public ChannelInitializer<? extends Channel> getChannelInitializer();

	public void setChannelInitializer(ChannelInitializer<? extends Channel> initializer);

	public NettyConfig getNettyConfig();
}

File: src/main/java/org/profit/server/NettyServer.kt
package org.profit.server

import io.netty.channel.Channel
import io.netty.channel.ChannelInitializer

/**
 * @author Kyia
 */
interface NettyServer : Server {
    var channelInitializer: ChannelInitializer<out Channel?>?
    val nettyConfig: NettyConfig
}